scylla-cdc-base/src/main/java/com/scylladb/cdc/cql/CQLConfiguration.java

```diff
@@ -9,13 +9,64 @@
 
 public class CQLConfiguration {
     private static final int DEFAULT_PORT = 9042;
+    private static final ConsistencyLevel DEFAULT_CONSISTENCY_LEVEL = ConsistencyLevel.QUORUM;
+
+    /**
+     * The consistency level of read queries to Scylla.
+     */
+    public enum ConsistencyLevel {
```

Comment:
what about EACH_QUORUM ?

Reply:
It's not supported for reads: https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/dml/dmlConfigConsistency.html#Table2.ReadConsistencyLevels

Reply:
OK. I didn't know about this. Thanks.